lib/miq_automation_engine/service_models/miq_ae_service_miq_request_task.rb

```diff
@@ -20,6 +20,14 @@ def finished(msg)
       object_send(:update_and_notify_parent, :state => 'finished', :message => msg)
     end
 
+    def set_canceling
+      object_send(:update_attributes, :cancelation_status => 'canceling')
+    end
+
+    def set_canceled
+      object_send(:update_attributes, :cancelation_status => 'canceled')
```

Comment:
Is this needed due to how v2v is controlling the cancel process?  If so this logic should still be code at the active_record layer and you just need to `expose` the method here.